seatunnel-connectors-v2/connector-tdengine/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Add this module into [here](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml#L232)